guacamole-docker/bin/start.sh

```diff
@@ -594,6 +594,46 @@ END
 
 }
 
+## Adds properties to guacamole.properties which select the SAML
+## authentication provider, and configure it to connect to the specified SAML
+## provider.
+##
```

Comment:
Please correct the style of this comment to match:

```
##
## This somewhat JavaDoc-like format that we've adopted for documenting shell
## script functions.
##
```

rather than:

```
## This, which is missing the leading JavaDoc-like "opening" blank line
##
```

You're just missing that initial empty `##` line.

Reply:
done